File: 백준/Bronze/17273. 카드 공장 （Small）/카드 공장 （Small）.py
import sys
input = sys.stdin.readline

N, M = map(int, input().split())
cards = []
for _ in range(N):
    a, b = map(int, input().split())
    cards.append([a, b])
    
current = [card[0] for card in cards]

for _ in range(M):
    K = int(input())
    for i in range(N):
        if current[i] <= K:
            if current[i] == cards[i][0]:
                current[i] = cards[i][1]
            else:
                current[i] = cards[i][0]

print(sum(current))

File: 백준/Bronze/17273. 카드 공장 （Small）/카드 공장 （Small）.js
const input = require('fs').readFileSync(0).toString().trim().split('\n');
const [N, M] = input.shift().split(' ').map(Number);
const cards = [];
for (let i = 0; i < N; i++) {
    const [a, b] = input.shift().split(' ').map(Number);
    cards.push([a, b]);
}

const current = cards.map(card => card[0]);

for (let i = 0; i < M; i++) {
    const K = Number(input.shift());
    
    for (let j = 0; j < N; j++) {
        if (current[j] <= K) {
            if (current[j] === cards[j][0]) {
                current[j] = cards[j][1]
            } else {
                current[j] = cards[j][0]
            }
        }
    }
}

console.log(current.reduce((acc, cur) => acc + cur, 0));